packages/webapp/pages/plus/index.tsx

```diff
@@ -45,9 +47,27 @@ const PlusPage = ({
   giftToUser,
   shouldShowPlusHeader,
 }: PlusPageProps): ReactElement => {
+  const { logEvent } = useLogContext();
   const { isReady } = useRouter();
   const isLaptop = useViewSize(ViewSize.Laptop);
 
+  useEffect(() => {
+    const onScroll = () => {
+      logEvent({
+        event_name: LogEvent.PageScroll,
+        extra: JSON.stringify({
+          scrollTop: window.scrollY,
+        }),
+      });
+    };
+
+    globalThis?.window?.addEventListener('scrollend', onScroll);
+
+    return () => {
+      globalThis?.window?.removeEventListener('scrollend', onScroll);
```

Comment:
We have to monitor when the scrolling ends rather than `scroll` event. Otherwise, we will get flooded by data with just a single scroll like below:

https://github.com/user-attachments/assets/4a4fe117-66aa-472d-8d27-087832c56ded

Reply:
Maybe some debouncing too?

Reply:
For the scrollends it is much straightforward. I thought of using debounce too but wasn't sure of what should be the right value.

Reply:
For the `scrollend` - I think debounce shouldn't be needed as seen below.

https://github.com/user-attachments/assets/41905962-bc39-47fa-b5d3-974718aec2a9